common/gcp-unified/src/main/java/ai/h2o/mojos/deploy/common/transform/unified/MojoFrameToResponseConverter.java

```diff
@@ -0,0 +1,117 @@
+package ai.h2o.mojos.deploy.common.transform.unified;
+
+import static java.util.Arrays.asList;
+import static java.util.Collections.emptyList;
+import static java.util.Collections.emptySet;
+
+import ai.h2o.mojos.deploy.common.rest.unified.model.Row;
+import ai.h2o.mojos.deploy.common.rest.unified.model.ScoreRequest;
+import ai.h2o.mojos.deploy.common.rest.unified.model.ScoreResponse;
+import ai.h2o.mojos.runtime.frame.MojoFrame;
+import com.google.common.base.Strings;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import java.util.UUID;
+import java.util.function.BiFunction;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * Converts the resulting predicted {@link MojoFrame} into the API response object {@link
+ * ScoreResponse}.
+ */
+public class MojoFrameToResponseConverter
```

Comment:
so I just thought of something. Maybe it makes sense to do like a method override here, that way there is less copy paste. I believe it would be something like:
```
public class MyClass {
  public void doSomething(myString string) {}
  public void doSomething(myString, myInt int) {}
}
```
https://www.geeksforgeeks.org/overloading-in-java/ might be useful... idk I hate the idea of so much copy paste